### PR TITLE
ci: check-pr-semantic-title.yml workflow

### PR DESCRIPTION
**Pull request by SergeyRoyt**
https://monday.monday.com/boards/3532714909/pulses/5561513517

So we can start already getting used to semantic titles in the PRs :)

### CODE REVIEW

File: .github/workflows/check-pr-semantic-title.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+name: Check PR semantic title
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
+    branches:
+      - master
```
Comment (YossiSaadi) [inline .github/workflows/check-pr-semantic-title.yml:12]: I guess we would want to keep semantic commits for all PRs, also if they are not targeted to get into `master`

File: .github/workflows/check-pr-semantic-title.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+permissions:
+  pull-requests: read
+
+jobs:
+  check-pr-semantic-title:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Validate PR title
+        uses: amannn/action-semantic-pull-request@v5
```
Comment (YossiSaadi) [inline .github/workflows/check-pr-semantic-title.yml:22]: Very nice tool!

**Review by YossiSaadi — APPROVED** [review]
Seems cool, very popular tool, let's try it for a while :)

**Comment by talkor** [comment]
@SergeyRoyt is it working also if you edit the title before merge? in the squash commit

**Pull request merged**